packages/gatsby-theme-carbon/src/components/markdown/AutolinkHeader.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import GHSlugger from 'github-slugger';
+import Link from '@carbon/icons-react/lib/link/20';
+import { css } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const slugger = new GHSlugger();
+
+const Anchor = styled.a`
```

Comment:
Can we put these styles in a sass file 😇

Reply:
className toggling is for squares!

Reply:
( I'll check it out :) )

Reply:
> Looking good! I think we might also need this for h4s (we have it linking h4s on Carbon) but we should double check w/ @jeanservaas on that

She said h1-h3 is actually preferred.